### DISCUSSION TITLE
AI DECK sample code for sending data from pc to esp32 and gap8?

### DISCUSSION THREAD

**Discussion by mvtkurd** [bitcraze/discussions#121] [Q&A]
Hi all, 

Is there any example about sending data from PC to GAP8 through wifi of esp32 ? or any sample code which has been not published as example ? except the aideck-gap8-bootloader example!

Best,

**Comment by evoggy** [comment]
Hi! Unfortunately there's no good example for this yet, except for the bootloader script. I'm working on integrating this into the cflib (including a few examples) but I'm not done yet.

**Reply by mvtkurd** [reply]
@evoggy thanks for reply. will it be available before summer vacation ?
and another question, is there any possibility to use cflib to transfer data (array) from python script to the ai deck with help of  PA or USB and vice versa ?

**Reply by evoggy** [reply]
Hopefully it will be available next week, but I can't make any promises.

Depending on how much time I have it might include this, but most probable is that it will only include CRTP over CPX and not the other way around.

**Reply by mvtkurd** [reply]
Hi @evoggy 
May I ask that did you update the examples ?

**Reply by krichardsson** [reply]
@evoggy is currently enjoying his summer vacation and as far as I can see he did unfortunately not add an example. He will be back in about 3 weeks, we will make sure he updates this discussion when he is back

**Comment by mvtkurd** [comment]
Hi @evoggy 

May I ask for an update about this discussion ?

**Reply by evoggy** [reply]
Hi! Unfortunately there's no examples yet. I pushed some code to the lib for running CRTP over CPX before my vacation, but it's been really busy since I came back.
